pkg/driver/mount.go

```diff
@@ -272,6 +259,23 @@ func (m *S3Mounter) Mount(bucketName string, target string,
 	return nil
 }
 
+func addOptionToEnvironmentVariables(optionName string, envName string, options []string, env []string) ([]string, []string) {
+	// optionName is passed to the driver as a mount option, but is passed to MP via env variable
+	// so we need to remove it from the options and add it to the env.
+	optionIdx := -1
+	for i, o := range options {
+		if strings.HasPrefix(o, optionName) {
+			optionIdx = i
+			break
+		}
+	}
+	if optionIdx != -1 {
+		env = append(env, strings.Replace(options[optionIdx], optionName, envName, 1))
```

Comment:
`Replace` is safe here because we just checked `HasPrefix`, right? May be worth a short comment.